packages/utilities/gel-styled-components/README.md

```diff
@@ -0,0 +1,37 @@
+# gel-styled-components &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-styled-components.svg)](https://www.npmjs.com/package/@bbc/gel-styled-components) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection of variable constants which can be imported into your application.
+
+## Installation
+
+```jsx
+npm install @bbc/gel-styled-components --save
+```
+
+## Usage
+
+```jsx
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from 'gel-styled-components/breakpoints';
+
+import { GEL_GUTTER_BELOW_600PX } from 'gel-styled-components/spacings';
+
+import { GEL_BREVIER } from 'gel-styled-components/typography';
+```
+
+## Exports
+
+`/breakpoints` - GEL breakpoints, as well as typography breakpoints. Including links to the relevant GEL pages.
+`/spacings` - GEL spacings and GEL Grid margins and gutters. Including links to the relevant GEL pages.
```

Comment:
`Including links to the relevant GEL pages`

...did you mean to put links here? 🤔 

If you're referring to the comments inside the files, they're not that useful to an outsider who has to dig into their node_modules. I'd add links in the README here 👍

Reply:
Done :)